addons/mail/static/tests/owl/component/thread_tests.js

```diff
@@ -82,7 +96,222 @@ QUnit.test('dragover files on thread with composer', async function (assert) {
     assert.ok(
         document.querySelector('.o_Composer_dropZone'),
         "should have dropzone when dragging file over the thread");
+});
+
+QUnit.only('message list desc order', async function (assert) {
+    assert.expect(5);
+    let amountOfCalls = 0;
+    let lastId = 10000;
+    await this.start({
+        async mockRPC(route, args) {
+            if (args.method === 'message_fetch') {
+                if (amountOfCalls > 4) {
```

Comment:
explain `4`, what are the 4 previous calls to `message_fetch`?

Reply:
In fact, we only need 2 and this will be called only twice normally.
I'll add some assert step in this to ensure that it's the case